assets/app/scripts/controllers/build.js

```diff
@@ -153,7 +143,8 @@ angular.module('openshiftConsole')
 
         $scope.cloneBuild = function() {
           var name = _.get($scope, 'build.metadata.name');
-          if (name && $scope.canBuild) {
+          $('.clone-build-btn').blur();
```

Comment:
go ahead and take this out, if we decide to do this we will do it everywhere at once